test/hotspot/jtreg/compiler/vectorapi/VectorFPtoIntCastTest.java

```diff
@@ -87,7 +87,7 @@ public VectorFPtoIntCastTest() {
 
     @Test
     @IR(counts = {IRNode.VECTOR_CAST_F2I, IRNode.VECTOR_SIZE_16, "> 0"},
-        applyIfCPUFeature = {"avx512f", "true"})
+        applyIfCPUFeatureOr = {"avx512f", "true", "avx10_2", "true"})
```

Comment:
You should check for target specific Machine IR which is selected on AVX10_2 targets.

Reply:
New checks are added.